package.json

```diff
@@ -7,5 +7,9 @@
     "start": "cd ./server && cargo run & cd ./client && yarn start-local",
     "build": "cd ./client && yarn build && cd ../server && cargo build --release",
     "test": "cd ./server && ulimit -n 1000 && cargo test && cd ../client && yarn test"
+  },
+  "dependencies": {
```

Comment:
these shouldn't be installed at this level - they're client only packages and need to be installed in the client app only. Ideally, just in the package they are required in, which is `common` for the `react-dropzone`; and `papaparse` is already installed, and does not need to come in again.

I'm going to remove directly in the branch, as the inclusion has blown out the PR files. Hope that's ok